components/sidebar/sidebar_channel_list/sidebar_channel_list.tsx

```diff
@@ -176,7 +176,7 @@ export default class SidebarChannelList extends React.PureComponent<Props, State
     }
 
     getFirstUnreadChannelFromChannelIdArray = (channelIds: string[]) => {
-        if (!this.props.currentChannelId) {
+        if (typeof this.props.currentChannelId !== 'string') {
```

Comment:
Could we maybe remove this block entirely? I can't see any reason that this would cause problems if currentChannelId is undefined or anything. I wonder if we were originally passing in `currentChannel` as an object and this was a null check before using `currentChannel.id` below

Reply:
Yeah, that's what it was. This code originally null checked `this.props.currentChannel` and after going back and forth a few times between a channel and its ID, we eventually settled on passing `currentChannelId`, so this check can just be removed.